src/components/atoms/InputText/__snapshots__/InputText.test.tsx.snap

```diff
@@ -361,15 +381,20 @@ exports[`<InputText /> renders the component with focus 1`] = `
   transition-property: border,box-shadow;
   -webkit-transition: 0.2s ease-in-out;
   transition: 0.2s ease-in-out;
+  font-family: "Open Sans",Roboto,Helvetica,Arial,sans-serif;
```

Comment:
Is this a breaking change? As in this affects the default theme if font-family was different (for whatever reason)?

Reply:
I didn't change this piece of Theme structure, the font was defined there previously, i just used it  `theme.typography.primary`